_sass/_base.scss

```diff
@@ -369,23 +369,58 @@ pre {
     font-weight: 400;
 
     @include large-desktop {
-      margin-left: 45px;
+      // margin-left: 45px;
     }
 
     @include small-desktop {
-      margin-left: 35px;
+      // margin-left: 35px;
       font-size: 16px;
     }
 
     @include tablet {
-      margin-left: 45px;
+      // margin-left: 45px;
     }
 
     @include phone {
-      margin-left: 35px;
-      font-size: 16px;
+      // margin-left: 35px;
+      font-size: 14px;
     }
   }
+  .center-logo{
+    height: 50px;
+    text-align: center !important;
+  }
+  
+  &.facebook {
+    @include button-colors($color-facebook);
+  }
+  &.reddit {
+    @include button-colors($color-reddit);
+  }
+  &.twitter {
+    @include button-colors($color-twitter);
+  }
+  &.youtube {
+    @include button-colors($color-youtube);
```

Comment:
I know there are some existing buttons for FB, TW, etc.  Just wanted to make sure you saw those and determined they couldn't be reused.